playbooks/adhoc/uninstall.yml

```diff
@@ -41,7 +42,7 @@
         - origin-node
 
     - yum: name={{ item }} state=absent
-      when: not is_atomic
+      when: is_atomic == "False"
```

Comment:
Ah, I see what it is...  the assignment is converting it to a string for some reason...  try this:
```
when: not is_atomic | bool
```